stability/stability/windows_native-stability-test.py

```diff
@@ -148,7 +148,7 @@ def runParsec():
     targetFile = os.path.join(targetDir, 'blackscholes.tar.gz')
     downloadAndUnpack(benchmarkLocation, targetFile)
     
-    return runAndProcess("%WORKSPACE%\\blackscholes_cpp_serial 1 in_10M.txt prices.txt", parsecProcessResults)
+    return runAndProcess("START \"STABILITY_PERF_RUN\" /B /WAIT /HIGH /AFFINITY 0x2 %WORKSPACE%\\blackscholes_cpp_serial 1 in_10M.txt prices.txt", parsecProcessResults)
```

Comment:
When I tested this locally it launches correctly.  This works because we are using shell=True so we get shell commands when we run the command line passed.